dbt/adapters/bigquery/gcloud.py

```diff
@@ -1,6 +1,6 @@
 from dbt.adapters.events.logging import AdapterLogger
-import dbt.common.exceptions
-from dbt.common.clients.system import run_cmd
+import dbt_common.exceptions
```

Comment:
Is there a benefit to pulling in all exceptions over the ones we use?

Reply:
good catch!